### DISCUSSION TITLE
format: Anyone getting duplicated time

### DISCUSSION THREAD

**Discussion by carolineoliva** [js-fns/date-fns#2746] [Q&A] [answered]
Hi guys!

We are seeing something strange when calling format function.
Sometimes, the result, for the same input, returns date and hour:minutes, and sometimes it returns time and time:minutes.

For instance:
out-17 14:54
17hs 14:54

This is how we are calling the format function: format(date, 'eeee', { locale: ptBR })
where date is a of type Date

Anyone else?

**Comment by fturmel** [comment, marked as answer]
@carolineoliva Could you share a more complete example? The `eeee` token should only be returning  the day of the week and not time related information.

https://date-fns.org/v2.25.0/docs/format

```js
import { format } from "date-fns";
import { ptBR } from "date-fns/locale";

const date = new Date(2021, 10, 3);
const res = format(date, "eeee", { locale: ptBR });

console.log(res); // quarta-feira
```

**Reply by carolineoliva** [reply]
@fturmel You're correct, I'm the one who is appending the time, still I'm randomly getting '17hs' when using 'eeee' to get the day of the week.

```
import { format } from "date-fns";
import { ptBR } from "date-fns/locale";

const date = new Date(2021, 10, 17, 14, 54, 0, 0);
const dayOfTheWeek = format(date, 'eeee', { locale: ptBR });
const time = format(date, 'HH:mm', { locale: ptBR });
console.log(dayOfTheWeek); // 17hs
console.log(`${dayOfTheWeek} ${preposition} ${time}`) // 17hs às 14:54 (expecting domingo às 14:54)
```
where preposition is either 'às' or 'à'

Thanks in advance!

**Reply by fturmel** [reply]
I can't replicate this behavior on my end. Which version of `date-fns` are you using?

Could you put together an example on https://codesandbox.io/ that reproduces the problem?

**Reply by carolineoliva** [reply]
I was using 2.23.0, now I have updated the dependency to 2.25.0. I didn't have problems since the update.
Let's see if it happens again. If it does, I'll put together the code that reproduces the problem.
Thank you very much for your assistance.
